src/pages/home/_markets.js

```diff
@@ -36,12 +35,12 @@ const MarketCard = styled.article`
     }
 `
 const Markets = () => (
-    <SectionContainer background="grey-10">
+    <SectionContainer style={{ backgroundColor: 'rgba(242, 243, 244, 0.3)' }}>
```

Comment:
do we allow to use inline styles?

Reply:
@arash-fs I personally don't mind inline styling if it's for adding one or two properties but it could be changed to a styled component too